### ISSUE TITLE
RUB-398 ⁃ Token preauthorisation issue

### ISSUE THREAD

**Issue by thatdoogieguy** [thatdoogieguy/MapMyBins#2]
A new user was able to create an account and token. But their credit card was refused, 05 Do Not Honor.

I think we will have to do the $1 preauthorisation for new tokens. 

What are you thoughts on this and how it will work with the sign up process?

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~colby]: first, need to inform user that their credit card will be charged $1 for authorization. There will be a new page or modal for waiting the authorization process done, mostly just a loading page, like when I pay something with my credit card. and when the authorization is done:

  * success -> thank you page
  * still error -> display error message.

For 05 Do Not Honour, from https://go.eway.io/s/article/Transaction-Response-Code-05-Do-not-Honour?language=en_US: "You can also contact your merchant bank for a more detailed explanation of why the transaction was declined. Alternatively, you can wait for an hour or so and try the card again as often the issue that caused the 05 error is temporary."

**Comment by thatdoogieguy** [comment]
➤ User  commented:

My bank has an App that allows you to lock your credit card when you misplace it. When I tried to buy something I got the 05 do not honor response. So I think it's a response for everything else. 

I think we should make the following changes:

  * Mention the $1 authorisation charge in the sign up and when they update their card details. 
  * If the $1 fails then they will get the payment update prompt. But the account trial will still work.
  * For an 05 response it should try again after one day, then notify CMS admin if it fails.

What are your thoughts?
